### ISSUE TITLE
Dukecon is not accessible (a11y)

### ISSUE THREAD

**Issue by annam002** [dukecon/dukecon_pwa#97]
I tested the accessibility of the dukecon app (after talking with Steffi). Here are the results:

# Setup
I checked https://latest.dukecon.org/javaland/2019/#/schedule
I tested the following things:
* Check conformity with WCAG guidelines. A lot of accessibility tools can do that automatically: Axe, Wave, Accessibility Insights, etc.
* Check keyboard navigation
* Check screen reader navigation (with NVDA)

# Findings
## Color contrast
The color contrast is generally really bad. Use an accessibility tool to check out the specific places that need to be improved. 

## Missing alt tags
The language flags and the search icon do not have an alt text.

## Keyboard navigation
1. The keyboard focus is really hard to see on some items
2. I cannot reach the filters (only the "reset" button)
3. I cannot reach the "go to top" buttons
4. You have to tab through all the talks in the page to get to the bottom, some way of skipping over the slots would be nice
5. I cannot reach the "mark as favourite"-button
6. I cannot reach the "switch language"-button
7. I cannot reach the "login"-button

## Search
1. The search input doesn't have a label
2. The search button is not a button, just an icon. I get that search is executed while you type, but the icon is really misleading. Maybe remove it, or move it into the search box.

## Screen reader specific stuff
1. Navigating the talks overview is hard: The timeslots are html tables, with the first column being the time, the second column the talks in the slot. With the shortcut 't' you can jump to the next timeslot, however, it is impossible to read a talk summary individually. It would be better if the talks would be in an html list, then it is possible to jump from talk to talk with keyboard shortcuts.
2. The "takes longer" info is not read
3. When using the search, there is no auditory feedback that the talk overview changed
4. There are no landmarks which identify the content as header, navigation, content, footer. Would make screen reader navigation easier.

**Comment by steffchep** [comment]
Should now be navigable by keyboard,
Still To do: alt texts for screen readers
